Wrappers/Python/cil/optimisation/algorithms/SPDHG.py

```diff
@@ -175,8 +175,8 @@ def update(self):
             y_k.multiply(self.sigma[i], out=y_k)
             y_k.add(self.y_old[i], out=y_k)
             
-        self.f[i].proximal_conjugate(y_k, self.sigma[i], out=y_k)
-        
+        tmp = self.f[i].proximal_conjugate(y_k, self.sigma[i])
+        y_k = tmp
```

Comment:
is there a reason why this could not happen in one line as 
`y_k = self.f[i].proximal_conjugate(y_k, self.sigma[i])`

Reply:
ah, nope :)